Dockerfile

```diff
@@ -1,17 +1,126 @@
-# This Dockerfile creates a production release image for the project using crt release flow.
-FROM alpine:3.13 as  default
+# This Dockerfile contains multiple targets.
+# Use 'docker build --target=<name> .' to build one.
+# e.g. `docker build --target=dev .`
+#
+# All non-dev targets have a VERSION argument that must be provided 
+# via --build-arg=VERSION=<version> when building. 
+# e.g. --build-arg VERSION=1.11.2
+#
+# `default` is the production docker image which cannot be built locally. 
+# For local dev and testing purposes, please build and use the `dev` docker image.
+
+
+# Official docker image that includes binaries from releases.hashicorp.com. This
+# downloads the release from releases.hashicorp.com and therefore requires that
+# the release is published before building the Docker image.
+FROM alpine:3.13 as official
```

Comment:
Should we be using our mirror for the base image? ie `docker.mirror.hashicorp.services/alpine:3.13.6`